docs/rules/no-disallowed-property-values.md

```diff
@@ -0,0 +1,30 @@
+# No Disallowed Property Values
+
+Rule `no-disallowed-property-values` will warn against the use of certain values for properties.
+
+## Options
+
+* `properties`: `{dictionary of properties mapped to array of disallowed values}` (defaults to empty dictionary `{}`).
+
+## Examples
+
+When `properties` contains a property value of `text-transform` as shown below:
+
+```yaml
+no-disallowed-property-values:
+  - 1
+  -
+    properties:
+      text-transform: ['capitalize']
+```
+
+The following would not be allowed:
+
+```scss
+
+// the capitalize value for text-transform is not allowed
+.foo {
+  text-transform: capitalize;
+}
+
```

Comment:
Can we just add a quick example of multiple property values also, just to be complete in how we allow the property lists to be defined.

Reply:
Done!